--- a/luci-theme-argon/ucode/template/themes/argon/header.ut
+++ b/luci-theme-argon/ucode/template/themes/argon/header.ut
@@ -47,7 +47,7 @@
 
 -%}
 <!DOCTYPE html>
-<html lang="{{ dispatcher.lang }}">
+<html lang="{{ dispatcher.lang }}"{% if (mode == 'dark'): %} data-darkmode="true"{% elif (mode == 'light'): %} data-darkmode="false"{% endif %}>
 
 <head>
 	{% include("themes/" + theme + "/head_meta", { hostname: hostname, node: node, bar_color: bar_color, media: media }) %}

--- a/luci-theme-argon/root/etc/config/argon
+++ b/luci-theme-argon/root/etc/config/argon
@@ -5,6 +5,6 @@ config global
 	option blur_dark '10'
 	option transparency '0.5'
 	option transparency_dark '0.5'
-	option mode 'normal'
+	option mode 'dark'
 	option online_wallpaper 'bing'

--- a/luci-theme-argon/ucode/template/themes/argon/footer.ut
+++ b/luci-theme-argon/ucode/template/themes/argon/footer.ut
@@ -19,13 +19,6 @@
 		</div>
 		<footer class="mobile-hide" style="text-wrap: auto">
 			<div class="footer-content" style="display: flex; flex-wrap: wrap; gap: 0.5em; justify-content: end;">
-				<a class="luci-link" href="https://github.com/openwrt/luci" target="_blank">Powered by {{ version.luciname }} ({{ version.luciversion }})</a>
-
-				<span class="footer-separator">|</span>
-
-				<a href="https://github.com/jerrykuku/luci-theme-argon" target="_blank">ArgonTheme {# vPKG_VERSION #}</a>
-
-				<span class="footer-separator">|</span>
 
 				<a class="luci-link" href="{{ version.disturl }}" target="_blank">{{ version.distname }} {{ version.distversion }}-{{ version.distrevision }}</a>
 
