--- /dev/null
+++ b/applications/luci-app-dockerman/root/etc/uci-defaults/luci-app-dockerman
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+uci -q batch <<-EOF >/dev/null
+	set uhttpd.main.script_timeout="360"
+	commit uhttpd
+EOF
+
+/etc/init.d/dockerd restart

--- a/luci-app-dockerman/htdocs/luci-static/resources/view/dockerman/configuration.js
+++ b/luci-app-dockerman/htdocs/luci-static/resources/view/dockerman/configuration.js
@@ -66,6 +66,7 @@ return L.view.extend({
 		o.value('https://docker.io');
 		o.value('https://ghcr.io');
 		o.value('https://hub-mirror.c.163.com');
+		o.value('https://ustc-edu-cn.mirror.aliyuncs.com');
 
 		o = s.taboption('globals', form.ListValue, 'log_level',
 			_('Log Level'),
