--- a/luci-app-netstat/files/usr/share/luci/menu.d/luci-app-netstat.json
+++ b/luci-app-netstat/files/usr/share/luci/menu.d/luci-app-netstat.json
@@ -7,7 +7,7 @@
       "path": "netstat/config"
     },
     "depends": {
-      "acl": [ "luci-app-netstat" ]
+      "acl": [ "luci-app-netstat1" ]
     }
   }
 }

--- a/luci-app-netstat/fMakefile
+++ b/luci-app-netstat/fMakefile
@@ -9,7 +9,7 @@ PKG_LICENSE:=GPL-3.0
 
 LUCI_TITLE:=NET Stats
 LUCI_DESCRIPTION:=This LuCI app provides net statistic functionality in a web interface.
-LUCI_DEPENDS:=+vnstat +luci-compat
+LUCI_DEPENDS:=+luci-compat
 
 include $(TOPDIR)/feeds/luci/luci.mk

